hadoop-ozone/ozone-manager/pom.xml

```diff
@@ -144,6 +144,24 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <version>0.9.11</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.ranger</groupId>
+      <artifactId>ranger-intg</artifactId>
+      <version>3.0.0-SNAPSHOT</version>
```

Comment:
IIRC the feature we need will also be released in Ranger 2.3.0. If there is a 2.3.0-SNAPSHOT jar, can we use that (and try it) instead?

Reply:
I see that version on Ranger's ranger-2.3 release branch but it looks like it is not being published
```
[ERROR] Failed to execute goal on project ozone-manager: Could not resolve dependencies for project org.apache.ozone:ozone-manager:jar:1.3.0-SNAPSHOT: Could not find artifact org.apache.ranger:ranger-intg:jar:2.3.0-SNAPSHOT in apache.snapshots.https (https://repository.apache.org/content/repositories/snapshots) -> [Help 1]
```

FYI the Ranger 2.3.0 release is being tracked in this Jira: https://issues.apache.org/jira/browse/RANGER-3710

Reply:
Sounds good to me. We can keep 3.0.0-SNAPSHOT for now.